testsupport/wait/member.go

```diff
@@ -216,7 +217,7 @@ func UntilUserAccountContainsCondition(expected toolchainv1alpha1.Condition) Use
 }
 
 // UntilUserAccountIsBeingDeleted returns a `UserAccountWaitCriterion` which checks that the given
-// USerAccount has the deletion timestamp set
+// UserAccount has the deletion timestamp set
 func UntilUserAccountIsBeingDeleted() UserAccountWaitCriterion {
```

Comment:
not used - can be deleted (can be done in the following PR :wink: )

Reply:
Good catch! Noted, will handle in a separate PR.